src/core/namespace.c

```diff
@@ -1137,29 +1137,22 @@ static int mount_procfs(const MountEntry *m, const NamespaceInfo *ns_info) {
                  * means we really don't want to use it, since it would affect our host's /proc
```

Comment:
hmm, is this logic in this commit correct? i mean, we don't check the result of umount_recursive() immediately, hence it might have failed with EPERM or so, and thus there might still be a mount point in place, which we cannot move.

hence this change doesn't look that obvious to me?

Reply:
Indeed...

Reply:
`path_is_mount_point()` is restored now. PTAL.